apps/docs/src/common/pages/get-started/prototype.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+        <h2>Get started</h2>
+          <ul className="plain">
+            <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+            <li><A href="/get-started/project">Starting a project for developers</A></li>
+          </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Get started</span>
+        Prototyping for designers
```

Comment:
We can probably just have:
> Prototyping

Reply:
If we do that, should we look at making it consistent with the production section's heading to say 'Production code'?